tests/src/test_cases/widgets/test_span.c

```diff
@@ -375,4 +375,108 @@ void test_spangroup_style_text_letter_space(void)
     TEST_ASSERT_EQUAL_SCREENSHOT("widgets/span_08.png");
 }
 
+#if LV_FONT_MONTSERRAT_24 && LV_FONT_MONTSERRAT_20
+void test_spangroup_get_span_coords(void)
+{
+    /* Initialize the active screen and create a new span group */
+    active_screen = lv_screen_active();
+    spangroup = lv_spangroup_create(active_screen);
+
+    const uint32_t span_count = 5;
+    lv_span_t * spans[span_count];
+
+    /* Set styles and properties for the span group */
+    lv_obj_set_style_outline_width(spangroup, 1, 0);
+    lv_spangroup_set_indent(spangroup, 20);
+    lv_spangroup_set_mode(spangroup, LV_SPAN_MODE_BREAK);
+    lv_obj_set_width(spangroup, 300);
+    lv_obj_set_style_pad_all(spangroup, 20, LV_PART_MAIN);
+
+    /* Create spans and set their properties */
+    spans[0] = lv_spangroup_new_span(spangroup);
+    lv_span_set_text(spans[0], "China is a beautiful country.");
+    lv_style_set_text_color(lv_span_get_style(spans[0]), lv_palette_main(LV_PALETTE_RED));
+    lv_style_set_text_decor(lv_span_get_style(spans[0]), LV_TEXT_DECOR_UNDERLINE);
+    lv_style_set_text_opa(lv_span_get_style(spans[0]), LV_OPA_50);
+
+    spans[1] = lv_spangroup_new_span(spangroup);
+    lv_span_set_text_static(spans[1], "good good study, day day up.");
+    lv_style_set_text_font(lv_span_get_style(spans[1]), &lv_font_montserrat_24);
+    lv_style_set_text_color(lv_span_get_style(spans[1]), lv_palette_main(LV_PALETTE_GREEN));
+
+    spans[2] = lv_spangroup_new_span(spangroup);
+    lv_span_set_text_static(spans[2], "LVGL is an open-source graphics library.");
+    lv_style_set_text_color(lv_span_get_style(spans[2]), lv_palette_main(LV_PALETTE_BLUE));
+
+    spans[3] = lv_spangroup_new_span(spangroup);
+    lv_span_set_text_static(spans[3], "the boy no name.");
+    lv_style_set_text_color(lv_span_get_style(spans[3]), lv_palette_main(LV_PALETTE_GREEN));
+    lv_style_set_text_font(lv_span_get_style(spans[3]), &lv_font_montserrat_20);
+    lv_style_set_text_decor(lv_span_get_style(spans[3]), LV_TEXT_DECOR_UNDERLINE);
+
+    spans[4] = lv_spangroup_new_span(spangroup);
+    lv_span_set_text(spans[4], "I have a dream that hope to come true.");
+    lv_style_set_text_decor(lv_span_get_style(spans[4]), LV_TEXT_DECOR_STRIKETHROUGH);
+
+    /* Refresh the span group mode and update layout */
+    lv_spangroup_refr_mode(spangroup);
+    lv_obj_update_layout(spangroup);
+
+    /* Define expected coordinates for testing */
+    const lv_span_coords_t test_coords[span_count] = {
```

Comment:
Note this warning.

Reply:
no warning in fact, that was a old code snippet 😁